src/common/Tile/style.js

```diff
@@ -17,43 +17,87 @@ export const Container = styled.div`
     height: auto;
     grid-template-columns: auto 1fr;
   }
+
+  ${(props) =>
+    props.people &&
+    css`
```

Comment:
As before, it can be in one line. Why this practice?

Reply:
Prettier again 😉